README.md

```diff
@@ -19,14 +23,21 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
 - [F-Droid](https://gitlab.com/fdroid/fdroidclient) - F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform. The client makes it easy to browse, install, and keep track of updates on your device.
 - [Slimefun](https://github.com/Slimefun/Slimefun4)  - Turn your Minecraft Spigot server into a modpack with over 500+ items added! **Tree planted for every approved Hacktoberfest PR**
 
+## JavaScript
+- [Cloudflare Status Worker](https://github.com/WalshyDev/cf-status-worker) - Get a message in your Discord server when Cloudflare has an issue. Deployed with Clouflare Workers
+
 ## Kotlin
 - [Valetudo Companion](https://github.com/Hypfer/valetudo-companion) -  Valetudo Companion App for Android
 
 ## Python
 - [F-Droid Server](https://gitlab.com/fdroid/fdroidserver/) - Tools for managing builds, indexes, updates, and deployments for F-Droid repositories.
 
-## JavaScript
-- [Cloudflare Status Worker](https://github.com/WalshyDev/cf-status-worker) - Get a message in your Discord server when Cloudflare has an issue. Deployed with Clouflare Workers
+## Vimscript
+ - [auto-pairs](https://github.com/LunarWatcher/auto-pairs) - Plugin for automatically inserting and deleting pairs
+ - [Dawn](https://github.com/LunarWatcher/Dawn) - Project template generator
+
+## Zsh
+ - [Oh Ny Zsh](https://github.com/ohmyzsh/ohmyzsh/)
```

Comment:
Seems like a typo, should be Oh My Zsh, right?

Reply:
Also please add a description

Reply:
aw shit, my bad